CSS/Style.css

```diff
@@ -120,4 +127,9 @@ justify-content: right;
   
   .information:hover .informationbtn {
 	background-color: #3e8e41;
+  }
+
+  #TitluDePresa{

```

Comment:
would recommend to use classes and ids naming in english only - all the projects might have maintainers from all around the world